app/src/main/java/com/example/bookwormadventuresdeluxe2/NotificationUtility/NotificationData.java

```diff
@@ -0,0 +1,21 @@
+package com.example.bookwormadventuresdeluxe2.NotificationUtility;
+
+/**
+ * Body of the notification
+ */
+public class NotificationData
```

Comment:
Is this class actually doing anything?

Reply:
I renamed this to NotificationSender. Pls see processNotification inside NotificationHandler for usage. The added comments should clarify.
![processNotification Inside NotificationHandler](https://user-images.githubusercontent.com/44795394/100084273-cc2bbf80-2e07-11eb-897f-5cc75492f152.PNG)